src/Servers/IIS/AspNetCoreModuleV2/OutOfProcessRequestHandler/forwardinghandler.cpp

```diff
@@ -55,6 +55,7 @@ FORWARDING_HANDLER::FORWARDING_HANDLER(
     LOG_TRACE(L"FORWARDING_HANDLER::FORWARDING_HANDLER");
 
     m_fWebSocketSupported = m_pApplication->QueryWebsocketStatus();
+    m_fSendConnectionCloseHeader = m_pApplication->QueryConfig()->QueryEnableConnectionClose()->Equals(L"true", 1);
```

Comment:
These names should both be updated.
```suggestion
    m_fForwardResponseConnectionHeader = m_pApplication->QueryConfig()->QueryForwardResponseConnectionHeader()->Equals(L"true", 1);
```